numba/tests/test_moved_modules.py

```diff
@@ -0,0 +1,21 @@
+"""Tests for moved modules and their redirection from old path
+"""
+from numba.tests.support import TestCase
+
+
+class TestMovedModule(TestCase):
+    """Testing moved modules in Q1 2020 but were decided to kept as public API
+    """
+    def tests_numba_types(self):
+        import numba.types
```

Comment:
Should e.g. `from numba.types import containers` be explicitly tested too?

Reply:
i can add a test. In general, I am only testing a few things because I am already checking for object identity. I don't think that's any reason for it to not work.

Reply:
test added in https://github.com/numba/numba/pull/5676/commits/2998661e6f33d8b437aadd70f0b0f08302524b0e